### PR TITLE
Corrections from misspell-fixer

### PR DESCRIPTION
**Pull request by EdwardBetts**
New corrections copied from the misspell-fixer project.

https://github.com/vlajos/misspell-fixer

### CODE REVIEW

**Comment by larsoner** [comment]
Sorry, more merge conflicts now

**Comment by EdwardBetts** [comment]
I've fixed the merge conflicts.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -207,6 +209,7 @@ accoustic->acoustic
 accout->account
 accouting->accounting
 accoutn->account
+accoutrements->accouterments
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `accoutrements` seems valid

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4640,6 +4793,8 @@ dergistration->deregistration
 deriair->derriere
 dericed->derived
 deriffed->derived
+derivate->derivative
+derivates->derivatives
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: These are legitimate words

**Comment by larsoner** [comment]
Thanks @EdwardBetts

**Pull request merged**